pkg/app/artifacts.go

```diff
@@ -43,12 +67,32 @@ func playbookOptionsFromFlag(artifactFlags ArtifactsCmdFlags) (*ansible.Playbook
 	if err != nil {
 		return nil, fmt.Errorf("failed to find absolute path for --container-images-dir %w", err)
 	}
+	passedUserArgs := map[string]interface{}{
+		"os_packages_local_bundle_file":  osPackagesBundleFile,
+		"containerd_local_bundle_file":   containerdBundleFile,
+		"pip_packages_local_bundle_file": pipPackagesBundleFile,
+		"images_local_bundle_dir":        containerImagesDir,
+	}
+
+	if err = mergeUserOverridesToMap(artifactFlags.Overrides, passedUserArgs); err != nil {
```

Comment:
since the `passedUserArgs` flags are marked as `required`, should they get overriden from the override file and extra-vars? 
We should either not mark them as `required` or give them highest precedence if they are `required` . wdyt?

Reply:
Agreed. This code block should be the last one and given highest precedence